Documents/institution-status.md

```diff
@@ -0,0 +1,20 @@
+## Institution status
+
+Every filing will store a `InstitutionStatus` object with the following structure:
+
+* `status`: `String`
+
+For example:
+
+```json
+      "institution": {
+      "id": "12345",
+      "name": "First Bank",
+      "status": "active"
```

Comment:
I was taking the json directly from the API format, which I think that this documentation should follow. If we want to change how the bank status is returned I think that would be the purview of another ticket and the docs could be changed in that PR to match the new API response.